transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -64,12 +64,14 @@
    Return value is the KMS schema override map as configured in AWS as
    CMR_KMS_SCHEMA_OVERRIDE_JSON."
   []
-  (try
-    (json/parse-string (config/kms-scheme-override-json) true)
-    (catch com.fasterxml.jackson.core.JsonParseException e
-      (error "Failed to parse Scheme Override JSON while loading KMS resource [%s]: %s",
-             (config/kms-scheme-override-json)
-             (.getMessage e)))))
+  (let [overrides (config/kms-scheme-override-json)]
+    (debug (format "Requested KMS schema overrides: %s." overrides))
```

Comment:
May just want to make it `info` instead of debug

Reply:
I was trying to be conservative, but I like the suggestion!